### ISSUE TITLE
Outdated ODOO_RELEASE variable in 12.0/Dockerfile and pip saying that phonenumbers not installed 

### ISSUE THREAD

**Issue by wulaaf** [odoo/docker#245]
The current 12.0/Dockerfile contains:
ARG ODOO_RELEASE=20190128

There's an issue that was resolved in February but is not passed on to the docker image yet as the images haven't been updated. Refer to pictures below.

1) Why do we not use the ARG ODOO_RELEASE=latest instead of having to update the release by every release date? 

2) Once docker container is up and running, it says that phonenumbers is not installed, please 'run pip install phonenumbers'

![Screenshot 2019-03-29 18 45 29](https://user-images.githubusercontent.com/22291306/55277000-10a64580-531d-11e9-915b-b0d04419ed54.png)
![Screenshot 2019-03-29 18 45 32](https://user-images.githubusercontent.com/22291306/55277001-10a64580-531d-11e9-8a02-0e5ee731b595.png)

**Comment by codeagencybe** [comment]
Depends on how you run your setup. 
Odoo is usually pretty late with updating the Docker image(s), last update was 31 Jan (currently +2 months out of sync)

If you can't wait for them to release new image, you can also build your own image with Dockerfile and update from that one. 
You can find it here (Odoo 12): https://github.com/odoo/docker/blob/master/12.0/Dockerfile

With Docker Hub free plan you can have 1 custom image included for free but limited build time (1 parallel build). 
Or you can have a look at Gitlab, that one can do it too.
Basically, all you have to do is copy the entire Dockerfile from the current image and re-use and let the build run.
It will fetch the latest nightly build from Github and you are done. 
Or you can also modify it to suit your own environment, install extra dependencies, modules, ... whatever you want.

https://www.howtoforge.com/tutorial/how-to-create-docker-images-with-dockerfile/

**Comment by nuelgoz** [comment]
I also think that the version of odoo ODOO_RELEASE =latest should be used, regardless of the docker settings

**Comment by metaminux** [comment]
@wulaaf Dockerfile is now using ARG ODOO_RELEASE=20190816 (as of b5f088de13927177263d6cbc5ebf7d39de4dcfa7 ) and phonenumbers is already mentioned in #168 so maybe you can close this issue.

IMHO, Using ODOO_RELEASE=latest would just move the problem to ODOO_SHA argument

@d-fence You seem to be the one that update ODOO_RELEASE and ODOO_SHA args, can you give us a hint on why and when the updates occur ?

**Comment by d-fence** [comment]
@metaminux Hello, I update the Docker file right after the Odoo Xp when the new Odoo version is released. I 'm currently working on it. For intermediate versions, I try to update them regularly or when major fixes appears.

**Comment by d-fence** [comment]
@wulaaf and @nuelgoz as @metaminux stated, "latest" cannot be used in ODOO_RELEASE as it would break the sha verification.
https://github.com/odoo/docker/blob/b5f088de13927177263d6cbc5ebf7d39de4dcfa7/12.0/Dockerfile#L65

Because the deb package is built every night.
